docker/images/dev-cli/Dockerfile

```diff
@@ -1,6 +1,12 @@
 # Creates an image for skycoin development
 FROM golang:1.11-stretch
 
+# Image labels
+LABEL "org.label-schema.name"="skycoindev-cli develop" \
+      "org.label-schema.description"="Skycoin cli develop image" \
```

Comment:
Should be `Docker image for with go, node and command line tools for Skycoin developers`